types/react-native/index.d.ts

```diff
@@ -466,6 +468,79 @@ export interface Insets {
     right?: number;
 }
 
+export interface PressableStateCallbackType {
+    pressed: boolean;
+}
+
+export interface PressableProps extends AccessibilityProps, Omit<ViewProps, 'style'> {
+    /**
+    * Called when a single tap gesture is detected.
+    */
+    onPress?: (event: GestureResponderEvent) => void;
+
+    /**
+     * Called when a touch is engaged before `onPress`.
+     */
+    onPressIn?: (event: GestureResponderEvent) => void;
+
+    /**
+    * Called when a touch is released before `onPress`.
+    */
+    onPressOut?: (event: GestureResponderEvent) => void;
+
+    /**
+     * Called when a long-tap gesture is detected.
+     */
+    onLongPress?: (event: GestureResponderEvent) => void;
+
+    /**
+     * Either children or a render prop that receives a boolean reflecting whether
+     * the component is currently pressed.
+     */
+    children: React.ReactNode | ((state: PressableStateCallbackType) => React.ReactNode),
+
+    /**
+     * Duration (in milliseconds) from `onPressIn` before `onLongPress` is called.
+     */
+    delayLongPress?: number;
+
+    /**
+     * Whether the press behavior is disabled.
+     */
+    disabled?: boolean;
+
+    /**
+     * Additional distance outside of this view in which a touch is considered a
+     * press before `onPressOut` is triggered.
+     */
+    pressRectOffset?: number;
```

Comment:
Heads up that I’m planning on changing this in core today and picking it into 63. Changing it to pressRetentionOffset to be consistent with the touchables and Text